openslides/agenda/static/templates/agenda/list-of-speakers-partial-management.html

```diff
@@ -10,15 +10,10 @@
       </button>
     </span>
     <span os-perms="agenda.can_manage">
-      <button ng-if="item.speaker_list_closed" ng-click="closeList(false)"
-          class="btn btn-sm btn-default">
-        <i class="fa fa-toggle-off"></i>
-        <translate>Closed</translate>
-      </button>
-      <button ng-if="!item.speaker_list_closed" ng-click="closeList(true)"
-          class="btn btn-sm btn-default">
-        <i class="fa fa-toggle-on"></i>
-        <translate>Open</translate>
+      <button ng-click="closeList(!item.speaker_list_closed)"
+          class="btn btn-sm" ng-class="item.speaker_list_closed ? 'btn-danger' : 'btn-default'">
+        <i class="fa" ng-class="item.speaker_list_closed ? 'fa-square' : 'fa-square-o'"></i>
```

Comment:
This empty box is some kind of useless.. Why not change the button text to 'open list of speakers'?

Reply:
Done.